python/mead/config/twpos.json

```diff
@@ -48,7 +48,7 @@
 		"dsz": 30,
 		"wsz": 30,
 		"type": "char-conv",
-		"cfiltsz": [1,2,3,4,5,7]
+		"cfiltsz": [3]
```

Comment:
why is this change in this PR?

Reply:
Agree this doesnt necessarily belong in the PR, it turns out that all these filters dont seem to really help performance on the current baseline model